DirectEmployers/MyJobs
# [PD-3996] Add template file retrieval

## What's this PR do?

Allows communication to be modified by dynamically retrieving templates used for generating content.

When a notification task is executed, it retrieves templates for the given communication by first checking for a `templates/notifications/<NameOfRule>/<CommunicationType>.html` file. If that does not exists, it loads the `templates/notifications/default/<CommunicationType>.html` file for the template.

Eventually, we'll also check the database for a company-specific overwrite of a given template type.

## Screenshots (if appropriate)

## Where should the reviewer start?

## How should this be manually tested?
In shell, manually create a notification task and use the `notification_task.send()` method. Ensure the console prints are what is expected.

## Any background context you want to provide?
There are some prints and test templates in this PR. They will be removed in the tickets that actually implement the email, sms, and myjobs mail tasks. For now, they help prove that this functionality works.
## PR Checklist

### Front-End


### Separation of Concerns


### Tests


### Migrations
